<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionArea</class>
 <widget class="QWidget" name="FunctionArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
  </property>
  <widget class="FunctionAreaWidget" name="widgetNewProject" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>44</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="FunctionAreaWidget" name="widgetOpen" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>44</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="FunctionAreaWidget" name="widgetUpload" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>44</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="FunctionAreaWidget" name="widgetSave" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>44</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSplitLine">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>13</y>
     <width>4</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/images/tool_area/TopToolBar_SplitLine.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FunctionAreaWidget</class>
   <extends>QWidget</extends>
   <header>FunctionAreaWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
